cpp/harvest/nacjd.cc

```diff
@@ -43,25 +43,31 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "marc_title_in_file new_marc_title_out_file [mode] [mode_params]\n"
-        "where marc_title_in_file contains also icpsr records (001 or 035a)\n"
+        "[mode] [mode_params]\n"
+        "get_full input_file output_file\n"
+        "\t- input_file: contains also icpsr records (001 or 035a)\n"
         "these records are not processed any more\n"
-        "new_marc_title_out_file contains all icpsr records not contained in input file.\n"
+        "\t- output_file contains all icpsr records not contained in input file.\n"
         "\n"
-        "marc_title_in_file new_marc_title_out_file count_of_rows number\n"
-        "\t- count_of_rows: mode name\n"
+        "get_by_count input_file output_file number\n"
+        "\t- input_file: contains also icpsr records (001 or 035a)\n"
+        "these records are not processed any more\n"
+        "\t- output_file contains all icpsr records not contained in input file.\n"
         "\t- number: is an optional parameter. If it is absent, the number of requests will be equal to 9000\n"
         "\n"
-        "marc_title_in_file new_marc_title_out_file get_by_ID ID\n"
-        "\t- get_by_ID: mode name\n"
-        "\t- ID: article identifier\n");
+        "get_by_ID output_file ID\n"
+        "\t- output_file contains all icpsr records not contained in input file.\n"
```

Comment:
The help text for output_file is incorrect. Since we have no input file here and we do not check for icpsr records in this mode, the output_file should contain the data for the single dataset.